### PR TITLE
irmin-pack: Disable the tests (flaky tests)

### PR DESCRIPTION
**Pull request by kit-ty-kate**
See https://github.com/ocaml/opam-repository/pull/20033#issuecomment-969965898
```
#=== ERROR while compiling irmin-pack.2.9.0 ===================================#
# context              2.1.1 | linux/x86_64 | ocaml-options-only-fp.1 | pinned(https://github.com/mirage/irmin/releases/download/2.9.0/irmin-2.9.0.tbz)
# path                 ~/.opam/4.13/.opam-switch/build/irmin-pack.2.9.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p irmin-pack -j 31
# exit-code            1
# env-file             ~/.opam/log/irmin-pack-2431-a9ad5e.env
# output-file          ~/.opam/log/irmin-pack-2431-a9ad5e.out
### output ###
#         test alias test/irmin-pack/runtest (exit 2)
# (cd _build/default/test/irmin-pack && ./test.exe -q --color=always)
# Testing `irmin-pack'.
# This run has ID `MZ14TDFH'.
# 
#   [OK]          PACK                      0   Basic operations on contents.
#   [OK]          PACK                      1   Basic operations on nodes.
#   [OK]          PACK                      2   Basic operations on commits.
#   [OK]          PACK                      3   Basic operations on branches.
#   [OK]          PACK                      4   Hash operations on trees.
#   [OK]          PACK                      5   Basic merge operations.
#   [OK]          PACK                      6   Basic operations on slices.
#   [OK]          PACK                      7   Test merges on tree updates.
#   [OK]          PACK                      8   Tree caches and hashconsing.
#   [OK]          PACK                      9   Complex histories.
#   [OK]          PACK                     10   Empty stores.
#   [OK]          PACK                     11   Private node manipulation.
#   [OK]          PACK                     12   High-level store operations.
#   [OK]          PACK                     13   High-level operations on trees.
#   [OK]          PACK                     14   High-level store synchronisation.
#   [OK]          PACK                     15   High-level store merges.
#   [OK]          PACK                     16   Unrelated merges.
#   [OK]          PACK                     17   Low-level concurrency.
#   [OK]          PACK                     18   Concurrent updates.
#   [OK]          PACK                     19   with_tree strategies.
#   [OK]          PACK                     20   Concurrent head updates.
#   [OK]          PACK                     21   Concurrent merges.
#   [OK]          PACK                     22   Shallow objects.
#   [OK]          PACK                     23   Closure with disconnected commits.
#   [OK]          PACK                     24   Clear.
#   [OK]          PACK                     25   Graph.Iter.
#   [OK]          PACK                     26   Watch.Callbacks and exceptions.
#   [OK]          PACK MEM                  0   Basic operations on contents.
#   [OK]          PACK MEM                  1   Basic operations on nodes.
#   [OK]          PACK MEM                  2   Basic operations on commits.
#   [OK]          PACK MEM                  3   Basic operations on branches.
#   [OK]          PACK MEM                  4   Hash operations on trees.
#   [OK]          PACK MEM                  5   Basic merge operations.
#   [OK]          PACK MEM                  6   Basic operations on slices.
#   [OK]          PACK MEM                  7   Test merges on tree updates.
#   [OK]          PACK MEM                  8   Tree caches and hashconsing.
#   [OK]          PACK MEM                  9   Complex histories.
#   [OK]          PACK MEM                 10   Empty stores.
#   [OK]          PACK MEM                 11   Private node manipulation.
#   [OK]          PACK MEM                 12   High-level store operations.
#   [OK]          PACK MEM                 13   High-level operations on trees.
#   [OK]          PACK MEM                 14   High-level store synchronisation.
#   [OK]          PACK MEM                 15   High-level store merges.
#   [OK]          PACK MEM                 16   Unrelated merges.
#   [OK]          PACK MEM                 17   Low-level concurrency.
#   [OK]          PACK MEM                 18   Concurrent updates.
#   [OK]          PACK MEM                 19   with_tree strategies.
#   [OK]          PACK MEM                 20   Concurrent head updates.
#   [OK]          PACK MEM                 21   Concurrent merges.
#   [OK]          PACK MEM                 22   Shallow objects.
#   [OK]          PACK MEM                 23   Closure with disconnected commits.
#   [OK]          PACK MEM                 24   Clear.
#   [OK]          PACK MEM                 25   Graph.Iter.
#   [OK]          PACK MEM                 26   Watch.Callbacks and exceptions.
#   [OK]          LAYERED_PACK              0   Basic operations on branches.
#   [OK]          LAYERED_PACK              1   Basic merge operations.
#   [OK]          LAYERED_PACK              2   Complex histories.
#   [OK]          LAYERED_PACK              3   Empty stores.
#   [OK]          LAYERED_PACK              4   Basic operations on slices.
#   [OK]          LAYERED_PACK              5   Private node manipulation.
#   [OK]          LAYERED_PACK              6   High-level store merges.
#   [OK]          LAYERED_PACK              7   Unrelated merges.
# > [FAIL]        LAYERED_PACK              8   Test commits and graphs.
```

### CODE REVIEW

**Comment by kit-ty-kate** [comment]
lower-bound failures with digestif 0.1
```
#=== ERROR while compiling irmin.2.0.0 ========================================#
# context              2.1.1 | linux/x86_64 | ocaml-base-compiler.4.06.1 | file:///home/opam/opam-repository
# path                 ~/.opam/4.06/.opam-switch/build/irmin.2.0.0
# command              ~/.opam/opam-init/hooks/sandbox.sh build dune build -p irmin -j 31
# exit-code            1
# env-file             ~/.opam/log/irmin-23-2f870c.env
# output-file          ~/.opam/log/irmin-23-2f870c.out
### output ###
#       ocamlc src/irmin/.irmin.objs/byte/irmin.{cmi,cmti} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlc.opt -w -40 -g -bin-annot -I src/irmin/.irmin.objs/byte -I /home/opam/.opam/4.06/lib/astring -I /home/opam/.opam/4.06/lib/base64 -I /home/opam/.opam/4.06/lib/bytes -I /home/opam/.opam/4.06/lib/digestif -I /home/opam/.opam/4.06/lib/fmt -I /home/opam/.opam/4.06/lib/jsonm -I /home/opam/.opam/4.06/lib/logs -I /home/opam/.opam/4.06/lib/lwt -I /home/opam/.opam/4.06/lib/ocamlgraph -I /home/opam/.opam/4.06/lib/re -I /home/opam/.opam/4.06/lib/result -I /home/opam/.opam/4.06/lib/sexplib -I /home/opam/.opam/4.06/lib/sexplib/0 -I /home/opam/.opam/4.06/lib/stringext -I /home/opam/.opam/4.06/lib/uchar -I /home/opam/.opam/4.06/lib/uri -I /home/opam/.opam/4.06/lib/uutf -no-alias-deps -open Irmin__ -o src/irmin/.irmin.objs/byte/irmin.cmi -c -intf src/irmin/irmin.mli)
# File "src/irmin/irmin.mli", line 1076, characters 19-29:
# Error: Unbound module type Digestif.S
#       ocamlc src/irmin/.irmin.objs/byte/irmin__Type.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlc.opt -w -40 -g -bin-annot -I src/irmin/.irmin.objs/byte -I /home/opam/.opam/4.06/lib/astring -I /home/opam/.opam/4.06/lib/base64 -I /home/opam/.opam/4.06/lib/bytes -I /home/opam/.opam/4.06/lib/digestif -I /home/opam/.opam/4.06/lib/fmt -I /home/opam/.opam/4.06/lib/jsonm -I /home/opam/.opam/4.06/lib/logs -I /home/opam/.opam/4.06/lib/lwt -I /home/opam/.opam/4.06/lib/ocamlgraph -I /home/opam/.opam/4.06/lib/re -I /home/opam/.opam/4.06/lib/result -I /home/opam/.opam/4.06/lib/sexplib -I /home/opam/.opam/4.06/lib/sexplib/0 -I /home/opam/.opam/4.06/lib/stringext -I /home/opam/.opam/4.06/lib/uchar -I /home/opam/.opam/4.06/lib/uri -I /home/opam/.opam/4.06/lib/uutf -intf-suffix .ml -no-alias-deps -open Irmin__ -o src/irmin/.irmin.objs/byte/irmin__Type.cmo -c -impl src/irmin/type.ml)
# File "src/irmin/type.ml", line 682, characters 12-29:
# Error: Unbound module Base64
#     ocamlopt src/irmin/.irmin.objs/native/irmin__Type.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlopt.opt -w -40 -g -I src/irmin/.irmin.objs/byte -I src/irmin/.irmin.objs/native -I /home/opam/.opam/4.06/lib/astring -I /home/opam/.opam/4.06/lib/base64 -I /home/opam/.opam/4.06/lib/bytes -I /home/opam/.opam/4.06/lib/digestif -I /home/opam/.opam/4.06/lib/fmt -I /home/opam/.opam/4.06/lib/jsonm -I /home/opam/.opam/4.06/lib/logs -I /home/opam/.opam/4.06/lib/lwt -I /home/opam/.opam/4.06/lib/ocamlgraph -I /home/opam/.opam/4.06/lib/re -I /home/opam/.opam/4.06/lib/result -I /home/opam/.opam/4.06/lib/sexplib -I /home/opam/.opam/4.06/lib/sexplib/0 -I /home/opam/.opam/4.06/lib/stringext -I /home/opam/.opam/4.06/lib/uchar -I /home/opam/.opam/4.06/lib/uri -I /home/opam/.opam/4.06/lib/uutf -intf-suffix .ml -no-alias-deps -open Irmin__ -o src/irmin/.irmin.objs/native/irmin__Type.cmx -c -impl src/irmin/type.ml)
# File "src/irmin/type.ml", line 682, characters 12-29:
# Error: Unbound module Base64
#       ocamlc src/irmin/.irmin.objs/byte/irmin__Hash.{cmi,cmti} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlc.opt -w -40 -g -bin-annot -I src/irmin/.irmin.objs/byte -I /home/opam/.opam/4.06/lib/astring -I /home/opam/.opam/4.06/lib/base64 -I /home/opam/.opam/4.06/lib/bytes -I /home/opam/.opam/4.06/lib/digestif -I /home/opam/.opam/4.06/lib/fmt -I /home/opam/.opam/4.06/lib/jsonm -I /home/opam/.opam/4.06/lib/logs -I /home/opam/.opam/4.06/lib/lwt -I /home/opam/.opam/4.06/lib/ocamlgraph -I /home/opam/.opam/4.06/lib/re -I /home/opam/.opam/4.06/lib/result -I /home/opam/.opam/4.06/lib/sexplib -I /home/opam/.opam/4.06/lib/sexplib/0 -I /home/opam/.opam/4.06/lib/stringext -I /home/opam/.opam/4.06/lib/uchar -I /home/opam/.opam/4.06/lib/uri -I /home/opam/.opam/4.06/lib/uutf -no-alias-deps -open Irmin__ -o src/irmin/.irmin.objs/byte/irmin__Hash.cmi -c -intf src/irmin/hash.mli)
# File "src/irmin/hash.mli", line 19, characters 17-27:
# Error: Unbound module type Digestif.S
```

**Comment by kit-ty-kate** [comment]
With digestif.0.6.1:
```
#=== ERROR while compiling irmin.2.0.0 ========================================#
# context     2.1.0~rc2 | linux/x86_64 | ocaml-base-compiler.4.06.1 | file:///home/opam/opam-repository
# path        ~/.opam/4.06/.opam-switch/build/irmin.2.0.0
# command     ~/.opam/4.06/bin/dune build -p irmin -j 47
# exit-code   1
# env-file    ~/.opam/log/irmin-6554-576a0c.env
# output-file ~/.opam/log/irmin-6554-576a0c.out
### output ###
# File "dune-project", line 4, characters 11-14:
# 4 | (using fmt 1.0)
#                ^^^
# Warning: Version 1.0 of integration with automatic formatters is not
# supported until version 1.4 of the dune language.
# There are no supported versions of this extension in version 1.1 of the dune
# language.
#
#       ocamlc src/irmin/.irmin.objs/byte/irmin__Hash.{cmo,cmt} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlc.opt -w -40 -g -bin-annot -I src/irmin/.irmin.objs/byte -I /home/opam/.opam/4.06/lib/angstrom -I /home/opam/.opam/4.06/lib/astring -I /home/opam/.opam/4.06/lib/base64 -I /home/opam/.opam/4.06/lib/bigarray-compat -I /home/opam/.opam/4.06/lib/bigstringaf -I /home/opam/.opam/4.06/lib/bytes -I /home/opam/.opam/4.06/lib/digestif -I /home/opam/.opam/4.06/lib/fmt -I /home/opam/.opam/4.06/lib/jsonm -I /home/opam/.opam/4.06/lib/logs -I /home/opam/.opam/4.06/lib/lwt -I /home/opam/.opam/4.06/lib/ocamlgraph -I /home/opam/.opam/4.06/lib/result -I /home/opam/.opam/4.06/lib/seq -I /home/opam/.opam/4.06/lib/stdlib-shims -I /home/opam/.opam/4.06/lib/stringext -I /home/opam/.opam/4.06/lib/uchar -I /home/opam/.opam/4.06/lib/uri -I /home/opam/.opam/4.06/lib/uutf -intf-suffix .ml -no-alias-deps -open Irmin__ -o src/irmin/.irmin.objs/byte/irmin__Hash.cmo -c -impl src/irmin/hash.ml)
# File "src/irmin/hash.ml", line 22, characters 43-58:
# Error: Unbound value H.to_raw_string
#     ocamlopt src/irmin/.irmin.objs/native/irmin__Hash.{cmx,o} (exit 2)
# (cd _build/default && /home/opam/.opam/4.06/bin/ocamlopt.opt -w -40 -g -I src/irmin/.irmin.objs/byte -I src/irmin/.irmin.objs/native -I /home/opam/.opam/4.06/lib/angstrom -I /home/opam/.opam/4.06/lib/astring -I /home/opam/.opam/4.06/lib/base64 -I /home/opam/.opam/4.06/lib/bigarray-compat -I /home/opam/.opam/4.06/lib/bigstringaf -I /home/opam/.opam/4.06/lib/bytes -I /home/opam/.opam/4.06/lib/digestif -I /home/opam/.opam/4.06/lib/fmt -I /home/opam/.opam/4.06/lib/jsonm -I /home/opam/.opam/4.06/lib/logs -I /home/opam/.opam/4.06/lib/lwt -I /home/opam/.opam/4.06/lib/ocamlgraph -I /home/opam/.opam/4.06/lib/result -I /home/opam/.opam/4.06/lib/seq -I /home/opam/.opam/4.06/lib/stdlib-shims -I /home/opam/.opam/4.06/lib/stringext -I /home/opam/.opam/4.06/lib/uchar -I /home/opam/.opam/4.06/lib/uri -I /home/opam/.opam/4.06/lib/uutf -intf-suffix .ml -no-alias-deps -open Irmin__ -o src/irmin/.irmin.objs/native/irmin__Hash.cmx -c -impl src/irmin/hash.ml)
# File "src/irmin/hash.ml", line 22, characters 43-58:
# Error: Unbound value H.to_raw_string
```

File: packages/irmin/irmin.2.0.0/opam [hunk 1]
```diff
@@ -15,13 +15,13 @@ build: [
 
 depends: [
   "ocaml"   {>= "4.06.0"}
-  "dune"    {>= "1.1.0"}
+  "dune"    {>= "1.2.0"}
   "fmt"     {>= "0.8.0"}
   "uri"     {>= "1.3.12"}
   "jsonm"   {>= "1.0.0"}
   "lwt"     {>= "2.4.7"}
   "base64"  {>= "2.0.0"}
```
Comment (mseri) [inline packages/irmin/irmin.2.0.0/opam:23]:
```suggestion
  "base64"  {>= "3.0.0"}
```
The module was renamed to `Base64` in version 3

**Comment by mseri** [comment]
We merge what we have now, we will fix the rest as it happens (if it does)

**Pull request merged**